ProcessMaker/modeler
# Fix start timer event

Fixes #284.

**How to test:**
* Drag a start timer event onto the gird.
* Change properties on the start timer event. Click away and click back.
* The properties on the start timer event should still be the ones you set.
